geopandas/tools/tests/test_sjoin.py

```diff
@@ -1,17 +1,36 @@
 from distutils.version import LooseVersion
+import math
+from typing import Sequence
+from geopandas.testing import assert_geodataframe_equal
 
 import numpy as np
 import pandas as pd
 
 from shapely.geometry import Point, Polygon, GeometryCollection
 
 import geopandas
-from geopandas import GeoDataFrame, GeoSeries, read_file, sjoin
+import geopandas._compat as compat
+from geopandas import GeoDataFrame, GeoSeries, read_file, sjoin, sjoin_nearest
+from geopandas.testing import assert_geoseries_equal
 
 from pandas.testing import assert_frame_equal
 import pytest
 
 
+MIN_PYGEOS_VERSION_FOR_NEAREST = "0.10"
```

Comment:
As per #2053, please use `PYGEOS_GE_010` instead.

Reply:
Done, I just copied it over